### PR TITLE
 LPS-121877 Migrate v2 usages in export-import-web

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Original pull https://github.com/moltam89/liferay-portal/pull/582

The <clay:management-toolbar-v2 /> tag has been replaced with the
<clay:management-toolbar /> tag (which uses clay v3).

**Test plan:** 
1. Navigate to Publishing > Export
2. Create a process and save. Here you see the management toolbar present in `toolbar.jsp`. 
3. On the top kebab menu, click on Export templates. Here is the management `toolbar from view_export_configurations.jsp`

I did not found how to render the changes done in `publish_templates/view.jsp`, so please test it, although, behaviour shouldn't have changed. 

Notice that there was a previous bug and it is still present in the PR: 

MT in export import

The total items are not being calculated. I guess that you are missing the `itemsTotal` parameter in the taglib. 

/cc @jonmak08 @julien @markocikos

### CODE REVIEW

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/CreationMenu.js [hunk 1]
```diff
@@ -42,6 +42,17 @@ const CreationMenu = ({
 		primaryItems.length + secondaryItemsCountRef.current
 	);
 
+	const getPlusIconLabel = () => {
+		const item =
+			primaryItems && primaryItems[0]
+				? primaryItems[0]
+				: secondaryItems[0].items
+				? secondaryItems[0].items[0]
+				: secondaryItems[0];
```
Comment (kresimir-coko) [inline modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/CreationMenu.js:51]: Just my 2 cents: this looks too concise, maybe for legibility in the future you could make it easier to read and follow what's going on. Not sure if my logic is spot on below but something in that vein.

```js
const firstPrimaryItem = primaryItems && primaryItems[0];
const firstSecondaryItem = secondaryItems[0].items ? secondaryItems[0].items[0] : secondaryItems[0];

const item = firstPrimaryItem || firstSecondaryItem;
```

**Review by kresimir-coko — APPROVED** [review]
Apart from the inline comment, looks simple enough of a change. 👍

File: modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/CreationMenu.js (hunk 1 above)
Comment (ambrinchaudhary) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/CreationMenu.js:51]: Good point @kresimir-coko, I just copied from the v2 (https://github.com/liferay/clay/blob/2.x/packages/clay-management-toolbar/src/ClayManagementToolbar.soy#L496) but that is for sure more legible.

Comment (ambrinchaudhary) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/CreationMenu.js]: Updated in  https://github.com/liferay-frontend/liferay-portal/pull/852/commits/4b524e02a25bcebf0ad1c14714727eab088243c4

Comment (wincent) [reply modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/CreationMenu.js]: Another option just for giggles:

```js
const item =
  primaryItems?.[0] ??
  secondaryItems[0].items?.[0] ??
  secondaryItems[0];
```

(Using `??` on the assumption that we're dealing with non-empty strings, but `||` would also be a viable option, depending on the context.)

File: modules/apps/export-import/export-import-web/src/main/resources/META-INF/resources/export/export_templates/view_export_configurations.jsp [hunk 2]
```diff
@@ -59,8 +59,8 @@ if (liveGroup == null) {
 ExportTemplatesToolbarDisplayContext exportTemplatesToolbarDisplayContext = new ExportTemplatesToolbarDisplayContext(request, liferayPortletRequest, liferayPortletResponse, liveGroupId, company, portletURL);
 %>
 
-<clay:management-toolbar-v2
-	displayContext="<%= exportTemplatesToolbarDisplayContext %>"
+<clay:management-toolbar
+	managementToolbarDisplayContext="<%= exportTemplatesToolbarDisplayContext %>"
```
Comment (jbalsas) [inline modules/apps/export-import/export-import-web/src/main/resources/META-INF/resources/export/export_templates/view_export_configurations.jsp:63]: The fact that we have nothing to change for these cases makes me quite happy! :)

**Review by jonmak08 — APPROVED** [review]

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
ci:stop

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:test:relevant

**Comment by ambrinchaudhary** [comment]
Forwarded to https://github.com/moltam89/liferay-portal/pull/588

**Pull request closed without merging**